CWE/inventions/culture_inventions.txt

```diff
@@ -629,117 +666,123 @@ social_anthropology = {
 }
 #cold war
 the_atomic_bomb = {
-        limit = { cold_war = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                        factor = 12
-                        OR = { 
+	limit = { cold_war = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
 				tag = USA 
 				tag = RUS 
 			} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 5
-            rebel_org_gain = {
-                faction = separatist_rebels
-                value = 0.50
-            }
-        }
+		}
+		modifier = {
+			factor = 2
+			is_greater_power = yes
+		}
+	}
+	effect = {
+		permanent_prestige = 5
+		rebel_org_gain = {
+		faction = separatist_rebels
+		value = 0.50
+		}
```

Comment:
Should also include rebel org gain for pro independence rebels

Reply:
It might just be me, but I'm kind of not sure why inventing the atomic bomb would increase rebel gain at all? Though I'll add it if you think it should.

Reply:
I see. In this case, I think it should be relocated to another tech that unlocks around the same time so as to keep separatist and pro independence rebels of the same strength as before this PR.

Reply:
I could keep it in there for now and add in multiple 'waves' of inventions later. It'd take time to go through the various Decolonial theories and national liberation schools to see which should be inventions.

Reply:
Alright